docs/source/en/model_doc/layoutlmv3.md

```diff
@@ -74,81 +200,66 @@ LayoutLMv3 is nearly identical to LayoutLMv2, so we've also included LayoutLMv2
 **Document question answering**
 - [Document question answering task guide](../tasks/document_question_answering)
 
-## LayoutLMv3Config
+## Model Details
```

Comment:
The rest of these changes should be reverted

Reply:
Not resolved yet, the `## Model Details` is still there as are the changes to the header levels of the LayoutLMv3 classes